modules/operator/helm.tf

```diff
@@ -26,14 +26,14 @@ resource null_resource "install_helm" {
 
   provisioner "file" {
     content     = data.template_file.install_helm.rendered
-    destination = "~/install_helm"
+    destination = "~/install_helm.sh"
   }
 
   provisioner "remote-exec" {
     inline = [
-      "chmod +x $HOME/install_helm",
-      "$HOME/install_helm",
-      # "rm -f $HOME/install_helm"
+      "chmod +x $HOME/install_helm.sh",
+      "$HOME/install_helm.sh",
+      # "rm -f $HOME/install_helm.sh"
```

Comment:
Now that you're proposing renaming each module script e.g. kubernetes module etc, maybe we should review 1. whether we need this script 2. rename it to something like install_helm_module?

Reply:
Agree but I'll keep that outside of this change